packages/admin/admin/src/hooks/useAsyncOptionsProps.ts

```diff
@@ -15,17 +15,21 @@ export function useAsyncOptionsProps<T>(loadOptions: () => Promise<T[]>): AsyncO
     const [loading, setLoading] = useState(false);
     const [error, setError] = useState<Error | null>(null);
 
+    useEffect(() => {
+        setLoading(true);
+        loadOptions()
+            .then((newOptions) => {
+                setOptions(newOptions);
```

Comment:
We need a ignore flag here as described in the docs: https://react.dev/reference/react/useEffect#fetching-data-with-effects.